Samples/kotlin/src/main/java/com/phunware/kotlin/sample/VoicePromptActivity.kt

```diff
@@ -1,63 +1,24 @@
 package com.phunware.kotlin.sample
 
-import android.animation.Animator
-import android.animation.AnimatorListenerAdapter
-import android.animation.AnimatorSet
-import android.animation.ObjectAnimator
 import android.content.Context
 import android.os.Bundle
 import android.speech.tts.TextToSpeech
-import android.support.design.widget.FloatingActionButton
-import android.support.design.widget.Snackbar
-import android.support.v7.app.AlertDialog
-import android.support.v7.app.AppCompatActivity
 import android.util.Log
 import android.view.View
-import android.widget.*
-import com.google.android.gms.maps.CameraUpdateFactory
-import com.google.android.gms.maps.model.LatLng
-import com.google.android.gms.maps.model.MapStyleOptions
-import com.phunware.core.PwCoreSession
-import com.phunware.core.PwLog
-import com.phunware.location.provider_managed.ManagedProviderFactory
-import com.phunware.location.provider_managed.PwManagedLocationProvider
-import com.phunware.location_core.PwLocationProvider
-import com.phunware.mapping.MapFragment
-import com.phunware.mapping.OnPhunwareMapReadyCallback
-import com.phunware.mapping.PhunwareMap
-import com.phunware.mapping.manager.Callback
+import android.widget.ImageButton
+import android.widget.TextView
 import com.phunware.mapping.manager.Navigator
-import com.phunware.mapping.manager.PhunwareMapManager
-import com.phunware.mapping.manager.Router
-import com.phunware.mapping.model.*
-import java.lang.ref.WeakReference
-import java.util.ArrayList
+import com.phunware.mapping.model.RouteOptions
 import java.util.Locale
 
-class VoicePromptActivity : AppCompatActivity(), OnPhunwareMapReadyCallback,
-        Building.OnFloorChangedListener, Navigator.OnManeuverChangedListener, TextToSpeech.OnInitListener {
+class VoicePromptActivity : WalkTimeActivity(), TextToSpeech.OnInitListener {
```

Comment:
VoicePromptActivity now extends WalkTimeActivity, so we can remove all of the map loading, routing, and floor switching logic.  VoicePromptActivity largely now just contains code demonstrating the scenario.